### DISCUSSION TITLE
在高版本中替换掉optifine，替换为其他更好的渲染、优化模组

### DISCUSSION THREAD

**Discussion by Onesade** [Meloong-Git/PCL#1275] [提问] [answered]
如题，https://forum.quiltmc.org/t/why-you-shouldnt-use-optifine/78，[Sodium](https://modrinth.com/mod/sodium), [Lithium](https://modrinth.com/mod/lithium), [Starlight](https://github.com/Spottedleaf/Starlight/releases), [FerriteCore](https://modrinth.com/mod/ferrite-core), [EntityCulling](https://www.curseforge.com/minecraft/mc-mods/entityculling), and [LazyDFU](https://modrinth.com/mod/lazydfu)

**Comment by WForst-Breeze** [comment, marked as answer]
1. 模组太多了，更新速度不一样，支持的模组加载器还得做个区分，更别提一个页面能不能塞下了……（见相似反馈 #311
2. Optifine 就算在高版本表现再不好，也起到了一定的优化作用，其高版本用户依旧有很多，直接替换并非明智之举
3. Mod 不考虑内置直接下载（还是请见 #311

感谢您的建议~

**Reply by Onesade** [reply]
那希望在MOD下载的curseforge页面下的MOD下提供一个独立的渲染优化MOD列表推荐，里面包含比较热门的fabrice渲染优化模组
毕竟那个页面的空间还有挺多的

**Reply by WForst-Breeze** [reply]
可以去 Issue 页面 提交 [新功能提案](https://github.com/Hex-Dragon/PCL2/issues/new?assignees=&labels=%C2%B7+%E6%96%B0%E5%8A%9F%E8%83%BD%2C%E6%96%B0%E5%8F%8D%E9%A6%88&template=feature.yml) 看看龙猫怎么想，但是我个人认为通过的可能性不太大……
